frontend/src/Pages/MembersOverview.tsx

```diff
@@ -60,13 +62,30 @@ export default function MembersOverview() {
         },
         {field: 'memberId', headerName: 'Mitgliedsnummer', width: 120},
     ];
+
+    function searchMembers(): Member[] {
+        return membersDB?.filter(member => {
+            return member.name.toLowerCase().includes(search.toLowerCase()) ||
```

Comment:
this is such a miniscule operation so I really doubt it even matters in terms of performance, but I believe it's better practice to just make a variable for lowerCaseSearch here and then use that, rather than calling .toLowerCase() each time?
```suggestion
            const lowerCaseSearch = search.toLowerCase();
            return member.name.toLowerCase().includes(lowerCaseSearch) ||
```